README.md

```diff
@@ -1,2 +1,94 @@
 # utPLSQL-maven-plugin
 A maven plugin for running Unit Tests with utPLSQL v3+
+
+
```

Comment:
Is this change making the maven plugin releasable/usable or is it still a work in progress?
I think we're missing information in the readme about it.
If it's a work-in-progress, can you please add this info at the top of readme, to avoid any confusion?

Reply:
It is a fully operative plugin with all requested changes implemented.

Reply:
We don't have access to the Travis account so it will possible to check the test status.